docs/source/welcome_to_sycamore/get_started.md

```diff
@@ -121,6 +119,50 @@ This will crawl and download the data from the specified website. If you import
 If you want to prepare you data with custom code, you can use a Jupyter notebook to iterate and test it. NEED LINK
```

Comment:
Fix the NEED LINK here?

Maybe just point people at the local development instructions which have all the bits for that.